test/quantization/model.py

```diff
@@ -12,6 +12,10 @@
 from torch.nn import functional as F
 
 def prepare_inputs_for_model(inps):
+    # this is because input from lm-eval is 2d
+    if input.dim() != 2:
```

Comment:
looks like this function isn't tested either. Since `input` isn't defined (but inps is) I assume this code doesn't run. Do we have tests that we run locally within torchao?